File: Company/question2.java
// Given an integer array Arr of size N the task is to 
// find the count of elements whose value is greater 
// than all of its prior elements.

// Arr[]={7,4,8,2,9}
//  output = 3.

import java.util.Scanner;

public class question2 {

    public static void main(String[] args) {
         Scanner sobj=new Scanner(System.in);
        int size=0;
        System.out.println("Enter size of an array:");
        size=sobj.nextInt();
        int arr[]=new int[size];
        System.out.println("Enter array elements:");
        for(int i=0;i<size;i++)
        {
            arr[i]=sobj.nextInt();
        }

        int icnt=0;
        int maxSoFar=Integer.MIN_VALUE;
        
        for(int i=0;i<arr.length;i++)
        {
            if(arr[i]>maxSoFar)
            {
                maxSoFar=arr[i];
                icnt++;
            }
          
        }

        System.out.println("count is:"+icnt);
    }
    
}

File: Lecture9_Array/Question2.cpp
// find sum of all elements of an array

#include<iostream>
using namespace std;

int Sum(int arr[],int size)
{
    
    int sum=0;
    for(int i=0;i<size;i++)
    {
        sum=sum+arr[i];
    }
    return sum;
}


int main()
{

    int size=0;
    cout<<"Enter size of an array:\n";
    cin>>size;

    int arr[size]={0};
    cout<<"Enter array elements:\n";
    for(int i=0;i<size;i++)
    {
        cin>>arr[i];
    }
    cout<<"Your enterd array elements:\n";
    for(int i=0;i<size;i++)
    {
        cout<<arr[i]<<" ";
    }
    

    int iret=Sum(arr,size);
    cout<<"Summation is:"<<iret<<"\n";

    return 0;
}